sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -291,25 +300,26 @@ MarginCalculationLauncher::launch() {
       }
     }
     if (!eventsIds.empty()) {
-      results_.push_back(LoadIncreaseResult());
-      idx = results_.size() - 1;
-      results_[idx].resize(eventsIds.size());
-      results_[idx].setLoadLevel(minVariation);
       SimulationResult result0;
       findOrLaunchLoadIncrease(loadIncrease, minVariation, minVariation, minVariation, marginCalculation->getAccuracy(), result0);
```

Comment:
We should pass the LoadIncreaseResult here and edit its internal result_ attribute instead of using a temporary  SimulationResult

Reply:
I didn't manage to find a test case in which the program enter this location of the code so maybe this part is not correct.

Reply:
This comment is general and is about all the calls to findOrLaunchLoadIncrease

Reply:
Does it mean we shouldn't pass a SimulationResult object to the MarginCalculationLauncher::findOrLaunchLoadIncrease method and pass a LoadIncreaseResult object instead ?

Reply:
yes

Reply:
But we don't need the scenarios results in MarginCalculationLauncher::findOrLaunchLoadIncrease() method. So passing a SimulationResult may be more logical.

Reply:
No but we use the loadIncrease result, which is now included inside the loadIncreaseResult class